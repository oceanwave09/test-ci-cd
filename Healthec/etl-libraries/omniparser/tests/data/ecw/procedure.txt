apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
280807|280807.17502.436659.4030|17502|17502|436659|2022-02-03|1033655741||PEDS Response form completed |4030|09/20/2016||PEDS response form|311226|Developmental screen |311225|Procedure|170670
280807|280807.17502.341226.4003|17502|17502|341226|2021-02-23|1285693960||Does your child enjoy being swung, bounced on your knee, etc?|4003|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4004|17502|17502|341226|2021-02-23|1285693960||Does your child take an interest in other children?|4004|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4005|17502|17502|341226|2021-02-23|1285693960||Does your child like climbing on things, such as up stairs?|4005|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4006|17502|17502|341226|2021-02-23|1285693960||Does your child enjoy playing peek-a-boo/hide-and-seek?|4006|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4007|17502|17502|341226|2021-02-23|1285693960||Does your child ever pretend, for example, to talk on the phone or take care of a doll or pretend other things?|4007|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4008|17502|17502|341226|2021-02-23|1285693960||Does your child ever use his/her index finger to point, to ask for something?|4008|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4009|17502|17502|341226|2021-02-23|1285693960||Does your child ever use his/her index finger to point, to indicate interest in something?|4009|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4010|17502|17502|341226|2021-02-23|1285693960||Can your child play properly with small toys without just mouthing, fiddling, or dropping them?|4010|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4011|17502|17502|341226|2021-02-23|1285693960||Does your child ever bring objects over to you to show you something?|4011|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4012|17502|17502|341226|2021-02-23|1285693960||Does your child look you in the eye for more than a second or two?|4012|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4013|17502|17502|341226|2021-02-23|1285693960||Does your child ever seem oversensitive to noise? |4013|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4014|17502|17502|341226|2021-02-23|1285693960||Does your child smile in response to your face or your smile?|4014|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4015|17502|17502|341226|2021-02-23|1285693960||Does your child imitate you?|4015|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4016|17502|17502|341226|2021-02-23|1285693960||Does your child respond to his/her name when you call|4016|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4017|17502|17502|341226|2021-02-23|1285693960||If you point to a toy across the room, does your child look at it?|4017|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4018|17502|17502|341226|2021-02-23|1285693960||Does your child walk?|4018|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4019|17502|17502|341226|2021-02-23|1285693960||Does your child look at things you are looking at|4019|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4020|17502|17502|341226|2021-02-23|1285693960||Does your child make unusual finger movements near his/her face?|4020|No||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4021|17502|17502|341226|2021-02-23|1285693960||Does your child try to attract your attention to his/her own activity|4021|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4022|17502|17502|341226|2021-02-23|1285693960||Have your ever wondered if your child is deaf?|4022|No||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4023|17502|17502|341226|2021-02-23|1285693960||Does your child understand what people say?|4023|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4024|17502|17502|341226|2021-02-23|1285693960||Does your child sometimes stare at nothing or wander with no purpose?|4024|No||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.341226.4025|17502|17502|341226|2021-02-23|1285693960||Does your child look at your face to check your reaction when faced with something unfamiliar?|4025|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.366950.4030|17502|17502|366950|2021-07-20|1801238258||PEDS Response form completed |4030|07/20/2021||PEDS response form|311226|Developmental screen |311225|Procedure|170670
280807|280807.17502.366950.4003|17502|17502|366950|2021-07-20|1801238258||Does your child enjoy being swung, bounced on your knee, etc?|4003|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.366950.4004|17502|17502|366950|2021-07-20|1801238258||Does your child take an interest in other children?|4004|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.366950.4005|17502|17502|366950|2021-07-20|1801238258||Does your child like climbing on things, such as up stairs?|4005|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
280807|280807.17502.366950.4006|17502|17502|366950|2021-07-20|1801238258||Does your child enjoy playing peek-a-boo/hide-and-seek?|4006|Yes||M-CHAT questionaire|311224|M-CHAT|311223|Procedure|170670
