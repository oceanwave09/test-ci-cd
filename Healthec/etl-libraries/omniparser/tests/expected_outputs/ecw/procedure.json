[
	{
		"answers": "09/20/2016",
		"apuid": "280807",
		"category_id": "311225",
		"category_name": "Developmental screen",
		"encounter_date": "2022-02-03",
		"encounter_id": "436659",
		"item_id": "311226",
		"item_name": "PEDS response form",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "61ebc95c440f43bdf39e9c90d78ce00d",
		"question": "PEDS Response form completed",
		"question_id": "4030",
		"record_id": "280807.17502.436659.4030",
		"row_id": "9fdc3ced-e172-4940-8df5-55564b64725a"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child enjoy being swung, bounced on your knee, etc?",
		"question_id": "4003",
		"record_id": "280807.17502.341226.4003",
		"row_id": "83f7aa70-06e7-4c5e-b03b-54cfb3eb56e1"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child take an interest in other children?",
		"question_id": "4004",
		"record_id": "280807.17502.341226.4004",
		"row_id": "2dc451ab-97f3-409d-8ba1-40fabb1e29f9"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child like climbing on things, such as up stairs?",
		"question_id": "4005",
		"record_id": "280807.17502.341226.4005",
		"row_id": "df68ec5d-5064-4883-8991-09c8cf6aa28c"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child enjoy playing peek-a-boo/hide-and-seek?",
		"question_id": "4006",
		"record_id": "280807.17502.341226.4006",
		"row_id": "eb2e136c-74dd-4908-bc80-ecb6edfddfe5"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child ever pretend, for example, to talk on the phone or take care of a doll or pretend other things?",
		"question_id": "4007",
		"record_id": "280807.17502.341226.4007",
		"row_id": "fc32e1e1-7f1f-4e38-b256-32459e53719a"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child ever use his/her index finger to point, to ask for something?",
		"question_id": "4008",
		"record_id": "280807.17502.341226.4008",
		"row_id": "7ea62497-190f-47a4-925d-e662b9d6c026"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child ever use his/her index finger to point, to indicate interest in something?",
		"question_id": "4009",
		"record_id": "280807.17502.341226.4009",
		"row_id": "e44619c0-9c96-4cab-a507-e841e535ea42"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Can your child play properly with small toys without just mouthing, fiddling, or dropping them?",
		"question_id": "4010",
		"record_id": "280807.17502.341226.4010",
		"row_id": "64ba02aa-eb64-4974-9ef3-03cfea139200"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child ever bring objects over to you to show you something?",
		"question_id": "4011",
		"record_id": "280807.17502.341226.4011",
		"row_id": "68f457b8-dec7-4887-ad32-71e52bbbb1d8"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child look you in the eye for more than a second or two?",
		"question_id": "4012",
		"record_id": "280807.17502.341226.4012",
		"row_id": "fedd402b-f3a8-4615-af60-34881eadb907"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child ever seem oversensitive to noise?",
		"question_id": "4013",
		"record_id": "280807.17502.341226.4013",
		"row_id": "45c08654-3fe9-47c8-992c-a238b23dc96c"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child smile in response to your face or your smile?",
		"question_id": "4014",
		"record_id": "280807.17502.341226.4014",
		"row_id": "3323749e-fbc8-4e94-9710-17c4592bbae5"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child imitate you?",
		"question_id": "4015",
		"record_id": "280807.17502.341226.4015",
		"row_id": "441b70f5-797e-4581-8e72-dd3d51953b68"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child respond to his/her name when you call",
		"question_id": "4016",
		"record_id": "280807.17502.341226.4016",
		"row_id": "4a74404a-c2f6-4290-bbab-97aec8064ee9"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "If you point to a toy across the room, does your child look at it?",
		"question_id": "4017",
		"record_id": "280807.17502.341226.4017",
		"row_id": "e3bf9bdc-215a-4633-a483-4e0304f0ac70"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child walk?",
		"question_id": "4018",
		"record_id": "280807.17502.341226.4018",
		"row_id": "54ee164d-1211-4b82-b374-1a913bf98520"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child look at things you are looking at",
		"question_id": "4019",
		"record_id": "280807.17502.341226.4019",
		"row_id": "5a8e6cc2-c496-494b-9ab8-c0af91de1904"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child make unusual finger movements near his/her face?",
		"question_id": "4020",
		"record_id": "280807.17502.341226.4020",
		"row_id": "7a3b2449-4c85-41c3-b241-ac5559a97b1f"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child try to attract your attention to his/her own activity",
		"question_id": "4021",
		"record_id": "280807.17502.341226.4021",
		"row_id": "7da7a793-79be-46e8-ad23-36b2379c60fb"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Have your ever wondered if your child is deaf?",
		"question_id": "4022",
		"record_id": "280807.17502.341226.4022",
		"row_id": "f9ae09d8-1075-4551-ad7f-43d07fa29dd5"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child understand what people say?",
		"question_id": "4023",
		"record_id": "280807.17502.341226.4023",
		"row_id": "8fd5fd32-ff9e-46e2-a4b0-772ed577d010"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child sometimes stare at nothing or wander with no purpose?",
		"question_id": "4024",
		"record_id": "280807.17502.341226.4024",
		"row_id": "7d24a559-4b01-4620-a684-9a53551ebfa1"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-02-23",
		"encounter_id": "341226",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "6435aaf3ad5b2f1fd862fe48da4dfff4",
		"question": "Does your child look at your face to check your reaction when faced with something unfamiliar?",
		"question_id": "4025",
		"record_id": "280807.17502.341226.4025",
		"row_id": "2b2bdc36-e776-4e27-b178-9633f93b50a4"
	},
	{
		"answers": "07/20/2021",
		"apuid": "280807",
		"category_id": "311225",
		"category_name": "Developmental screen",
		"encounter_date": "2021-07-20",
		"encounter_id": "366950",
		"item_id": "311226",
		"item_name": "PEDS response form",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e06f56eaf009b7bbae447dfab2ce587d",
		"question": "PEDS Response form completed",
		"question_id": "4030",
		"record_id": "280807.17502.366950.4030",
		"row_id": "30ccec21-17c3-4acb-85a7-7eb0361e4525"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-07-20",
		"encounter_id": "366950",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e06f56eaf009b7bbae447dfab2ce587d",
		"question": "Does your child enjoy being swung, bounced on your knee, etc?",
		"question_id": "4003",
		"record_id": "280807.17502.366950.4003",
		"row_id": "b7050728-4df6-4499-8a3a-e0c99603e11c"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-07-20",
		"encounter_id": "366950",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e06f56eaf009b7bbae447dfab2ce587d",
		"question": "Does your child take an interest in other children?",
		"question_id": "4004",
		"record_id": "280807.17502.366950.4004",
		"row_id": "fb3a4a83-4dde-4897-a76d-04807c028437"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-07-20",
		"encounter_id": "366950",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e06f56eaf009b7bbae447dfab2ce587d",
		"question": "Does your child like climbing on things, such as up stairs?",
		"question_id": "4005",
		"record_id": "280807.17502.366950.4005",
		"row_id": "aca528d5-cc62-433e-b1f0-6c0d93fbc9e2"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "311223",
		"category_name": "M-CHAT",
		"encounter_date": "2021-07-20",
		"encounter_id": "366950",
		"item_id": "311224",
		"item_name": "M-CHAT questionaire",
		"main_category_id": "170670",
		"main_category_name": "Procedure",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "e06f56eaf009b7bbae447dfab2ce587d",
		"question": "Does your child enjoy playing peek-a-boo/hide-and-seek?",
		"question_id": "4006",
		"record_id": "280807.17502.366950.4006",
		"row_id": "85435c38-09dc-40c7-a5ab-94f527efcb96"
	}
]
